### PR TITLE
Opengl rasterization for union

### PR DESCRIPTION
**Pull request by KaiSzuttor**

### CODE REVIEW

**Review by fweik — APPROVED** [review]

File: src/core/shapes/Union.hpp [hunk 1]
```diff
@@ -66,6 +67,14 @@ class Union : public Shape {
                         dist_compare);
   }
 
+  bool is_inside(Utils::Vector3d const &pos) const override {
```
Comment (KaiSzuttor) [inline src/core/shapes/Union.hpp:71]: @fweik this override is necessary if the union shape is used in a LB boundary.

File: src/core/shapes/Union.hpp [hunk 2]
```diff
@@ -66,6 +67,14 @@ class Union : public Shape {
...
+    return boost::accumulate(
```
Comment (fweik) [inline src/core/shapes/Union.hpp]: I think you should use `any_of`, which will short-cut and makes the intent clearer.

**Review by fweik — CHANGES_REQUESTED** [review]

File: src/core/shapes/Union.hpp (hunk 2 above)
Comment (KaiSzuttor) [reply src/core/shapes/Union.hpp]: you probably mean ```all_of```

**Comment by KaiSzuttor** [comment]
i will provide a test for LB boundary using the union shape which will make codecov happy

**Review by fweik — APPROVED** [review]

**Pull request merged**